pkg/scanner/creds/creds.go

```diff
@@ -0,0 +1,8 @@
+package creds
+
+import batchv1 "k8s.io/api/batch/v1"
+
+type CredentialAdder interface {
+	ShouldAdd() bool
```

Comment:
What is this method for? It also doesn't receive arguments so not clear what's it for..